core/blockchain_leader_rotation.go

```diff
@@ -0,0 +1,8 @@
+package core
+
+type stored struct {
```

Comment:
Can we change the name of this structure to be a bit more descriptive?

Reply:
Fixed. Thanks